packages/calling/src/CallingClient/registration/register.test.ts

```diff
@@ -732,7 +732,7 @@ describe('Registration Tests', () => {
       expect(reg.getStatus()).toBe(RegistrationStatus.INACTIVE);
       expect(reg.keepaliveTimer).not.toBe(timer);
 
-      webex.request.mockRejectedValueOnce(failurePayload).mockResolvedValue(successPayload);
+      webex.request.mockResolvedValue(successPayload);
```

Comment:
How does this change ensure we are failing over immediately? I'm not clear about how this statement change covers the new flow.

Reply:
The failure payload was for the primary server try but now we don't try primary server for the cc flow when it comes from the keepalive failure, here we show the success for the secondary server immediately.